bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/TypeConverter.java

```diff
@@ -506,6 +507,12 @@ private static void addErrorMessage(int errorCountDifference, List<String> error
         }
     }
 
+    private static boolean integerSubtypeMembers(Type inputValueType, Type targetType) {
```

Comment:
Shall we rename this method something like `isIntegerSubtypeMember()` or `checkIntegerSubtypeMember()` ? It is preferred the method name to be a verb.